src/components/TokenClaimDialog.tsx

```diff
@@ -58,7 +66,8 @@ export default function TokenClaimDialog({
   const rewardBalances = useContext(RewardsBalancesContext)
   const {
     claimsStatuses,
-    claimPoolReward,
+    // claimPoolReward,
```

Comment:
Is this just comment?

Reply:
This will be deleted. I will revisit this depending on if we want to keep that code for any reason or not.